frontend/javascripts/admin/statistic/time_tracking_overview.tsx

```diff
@@ -0,0 +1,261 @@
+        startDate.valueOf(),
+        endDate.valueOf(),
+        filteredTeams,
+        filteredProjects,
+      );
+      const filteredEntries: TimeEntry[] = await Request.receiveJSON(timeEntriesURL);
+      setIsFetching(false);
+      return filteredEntries;
+    },
+    allTimeEntries,
+    [
+      selectedTeams,
+      selectedTypes,
+      selectedProjectIds,
+      startDate,
+      endDate,
+      allTimeEntries,
+      mayUserAccessView,
+    ],
+  );
+  const filterStyle = { marginInline: 10 };
+
+  const exportToCSV = () => {
+    if (filteredTimeEntries == null || filteredTimeEntries.length === 0) {
+      return;
+    }
+    const timeEntriesAsString = filteredTimeEntries
+      .map((row) => {
+        return [
+          row.user.id,
+          row.user.firstName,
+          row.user.lastName,
+          Math.round(row.timeMillis / 1000),
+        ]
+          .map(String) // convert every value to String
+          .map((v) => v.replaceAll('"', '""')) // escape double quotes
+          .map((v) => (v.includes(",") || v.includes('"') ? `"${v}"` : v)) // quote it if necessary
+          .join(","); // comma-separated
+      })
+      .join("\n"); // rows starting on new lines
+    const csv = [TIMETRACKING_CSV_HEADER, timeEntriesAsString].join("\n");
+    const filename = "timetracking-export.csv";
+    const blob = new Blob([csv], {
+      type: "text/plain;charset=utf-8",
+    });
+    saveAs(blob, filename);
+  };
+
+  const rangePresets: TimeRangePickerProps["presets"] = [
+    { label: "Last 7 Days", value: [dayjs().subtract(7, "d").startOf("day"), currentTime] },
+    { label: "Last 30 Days", value: [dayjs().subtract(30, "d").startOf("day"), currentTime] },
+  ];
+
+  return (
+    <Card
+      title={"Annotation Time per User"}
+      style={{
+        marginTop: 30,
+        marginBottom: 30,
+      }}
+    >
+      <FilterOutlined />
+      <ProjectAndAnnotationTypeDropdown
+        setSelectedProjectIds={setSelectedProjectIds}
+        selectedProjectIds={selectedProjectIds}
+        setSelectedAnnotationType={setSelectedTypes}
+        selectedAnnotationType={selectedTypes}
+        style={{ ...filterStyle }}
+      />
+      <Select
+        mode="multiple"
+        placeholder="Filter teams"
+        defaultValue={[]}
+        style={{ width: 200, ...filterStyle }}
+        options={allTeams.map((team) => {
+          return {
+            label: team.name,
+            value: team.id,
+          };
+        })}
+        value={selectedTeams}
+        onSelect={(teamIdOrKey: string) => setSelectedTeams([...selectedTeams, teamIdOrKey])}
+        onDeselect={(removedTeamId: string) => {
+          setSelectedTeams(selectedTeams.filter((teamId) => teamId !== removedTeamId));
+        }}
+      />
+      <RangePicker
+        style={filterStyle}
+        value={[startDate, endDate]}
+        presets={rangePresets}
+        // TODO fix type error. see time_line_view: type error is commented out.
```

Comment:
todo

Reply:
Do you have an idea how to do this? The problem is that antd's dayjs and dayjs arent fully matching. Didn't really find a solution online, so I think about just ignoring the type checker there :sweat_smile: 

`Type '(dates: null | (Dayjs | null)[]) => void' is not assignable to type '(dates: NoUndefinedRangeValueType<Dayjs>, dateStrings: [string, string]) => void'.
  Types of parameters 'dates' and 'dates' are incompatible.
    Type 'NoUndefinedRangeValueType<Dayjs>' is not assignable to type '(Dayjs | null)[]'.
      Type 'import("[...]webknossos/node_modules/antd/node_modules/dayjs/index").Dayjs | null' is not assignable to type 'import("[...]/webknossos/node_modules/dayjs/index").Dayjs | null'.
        Type 'Dayjs' is missing the following properties from type 'Dayjs': fromNow, from, toNow, to, and 6 more.ts(2322)`

Reply:
I managed to fix it by changing the dayjs import to `import dayjs, { Dayjs } from "antd/node_modules/dayjs";`. Not sure whether thats clean though.

Reply:
sounds good to me :)